install.ps1

```diff
@@ -766,25 +797,89 @@ if (-not $noGui.IsPresent) {
 
     # Function that unchecks all the checkboxes
     function Clear-AllPackages {
-	foreach ($checkBox in $checkboxesPackages){
+	    foreach ($checkBox in $checkboxesPackages){
             $checkBox.Checked = $false
         }
+        $additionalPackagesBox.Items.clear()
     }
 
-    # Funtion that returns an array of packages that belong to a specific category
-    function Get-PackagesByCategory{
+    # Function that adds a new package to the listBox of additional packages
+    function Add-NewPackage {
         param (
-         [string]$category
+        [Parameter(Mandatory=$true)]
+        [string]$packageName
         )
-        return $packagesByCategory[$category]
+
+        if ($packageName -notin $listedPackages)
+        {
+           $additionalPackagesBox.Items.Add($packageName) | Out-Null
+           return $true
+        }
+        else{
+           return $false
+        }
+
+    }
+
+    function Get-ChocoPackage {
+        param (
+        [Parameter(Mandatory=$true)]
+        [string]$PackageName
+        )
+
+        choco search $PackageName -e -r | ForEach-Object {
+            $Name, $Version = $_ -split '\|'
+            New-Object -TypeName psobject -Property @{
+                'Name' = $Name
+                'Version' = $Version
+            }
+        }
+    }
+
+     function Get-VMPackage {
+        param (
+        [Parameter(Mandatory=$true)]
+        [string]$PackageName
+        )
+        if ($PackageName -notlike "*.vm") {
+            $PackageName = $PackageName + ".vm"
+        }
+        choco search $PackageName --exact -r -s "https://www.myget.org/F/vm-packages/api/v2" | ForEach-Object {
+            $Name, $Version = $_ -split '\|'
+            New-Object -TypeName psobject -Property @{
+                'Name' = $Name
+                'Version' = $Version
+            }
+        }
+    }
+
+    function Set-AdditionalPackages {
+        $additionalPackagesBox.Items.Clear()
+        foreach($package in $additionalPackages)
+        {
+            $additionalPackagesBox.Items.Add($package) | Out-Null
+        }
+    }
+
+    function Remove-SelectedPackages {
+        $additionalPackagesBox.BeginUpdate()
+        while ($additionalPackagesBox.SelectedItems.count -gt 0) {
+            $additionalPackagesBox.Items.RemoveAt($additionalPackagesBox.SelectedIndex)
+        }
+        $additionalPackagesBox.EndUpdate()
     }
 
     Add-Type -AssemblyName System.Windows.Forms
     [System.Windows.Forms.Application]::EnableVisualStyles()
+    $excludedCategories=@('Command and Control','Credential Access','Exploitation','Forensic','Lateral Movement', 'Payload Development','Privilege Scalation','Reconnaissance','Wordlists')
```

Comment:
I think we need to exclude `Web Application` as well. :thinking:

Reply:
[discussion] I am not sure if it is better to have a list of excluded categories or a list of categories. We have to main the list in either case. @mandiant/flare-vm opinions?